README.md

```diff
@@ -56,3 +56,454 @@ The `pages/api` directory is mapped to `/api/*`. Files in this directory are tre
 ## Deploy on Vercel
 
 Coming soon
+
+
+## **SDK Usage**
+
+
+### **All Pools**
+
+#### Pools List (Get All Pools On Current Chain)
+
+import { usePoolsData } from '@hooks/usePoolsData';
+
+//returns all pools on Midas & Relevant Data
+const { data: allPools } = usePoolsData()
+
+#### Tvl Locked
+	import { useSDK } from '@context/SDKContext';
+  import { BigNumber, utils } from 'ethers';
+
+  //returns TVL in Big Number format
+	const tvlNative = await useSDK().sdk.getTotalValueLocked(false)
+
+  //returns TVL in ETH value
+  utils.formatUnits(tvlNative)
+
+
+### **Asset Addition**
+
+#### Add Asset To Existing Pools (That You Own)
+
+    import { MarketConfig } from '@midas-capital/sdk';
+    import { usePoolData } from '@hooks/usePoolData';
+
+    const { data: poolData } = usePoolData(poolId);
+
+//TODO NOTE THE MARKETCONFIG TYPE IS NOT DOCUMENTED ANYWHERE ELSE
```

Comment:
I will give you access to https://docs.midascapital.xyz/developers/midas-sdk -> there we have a section on the types, so you can add the types you need and that are not here. We should link the types from here to the docs, so we don't duplicate them